config/locales/blacklight.en.yml

```diff
@@ -5,10 +5,10 @@ en:
       form:
         search:
           label: 'search for'
-          placeholder: ''
+          placeholder: 'Try "groundwater"'
```

Comment:
I find this helper tip a little confusing -- are there other suggestions?

Reply:
perhaps just "groundwater" or leave it blank entirely?

Reply:
how about "Search for data and maps"? or blank is ok with me. The former begs the question whether we need that header. This isn't my forte though :).

Reply:
Yep I just stole this from Airbnb:

![screen shot 2018-01-24 at 12 25 05 pm](https://user-images.githubusercontent.com/1656824/35352578-a51569cc-0101-11e8-8329-273d7db9a73b.png)

I'm inclined to leave it as is.. and see if we get any feedback. It's a quick change if we have to modify it.